CMakePresets.json

```diff
@@ -2,19 +2,148 @@
   "version": 3,
   "configurePresets": [
     {
-      "name": "Ninja",
-      "generator": "Ninja Multi-Config",
+      "name": "default",
+      "hidden": true,
+      "generator": "Ninja",
+      "binaryDir": "out/build/${presetName}",
+      "cacheVariables": {
+        "Languages_Description": "Modify the setting to any language you want: CHINESE;DUTCH;ENGLISH;FRENCH;GERMAN;ITALIAN;POLISH;RUSSIAN",
+        "Languages": "ENGLISH",
+        "CMAKE_RUNTIME_OUTPUT_DIRECTORY_Description": "Point the setting to your game directory, so you can run/debug the executables directly from there",
+        "CMAKE_RUNTIME_OUTPUT_DIRECTORY": "."
```

Comment:
here i have
```
C:\\Games\\Jagged Alliance 2
```
and it works for VS and CLion, they can directly let it run from there with ddraw.dll loading and debugging/breakpoints working (at least for me)